stdlib/2and3/mmap.pyi

```diff
@@ -0,0 +1,81 @@
+# Stubs for mmap
+
+from typing import (
+    Optional, Sequence, Union,
+    Generic, TypeVar, overload
+)
```

Comment:
Done in 82fff84; it is simply the way I'm writing it, separate type object-specification from types metadata. For `Type`, I was thinking of leaving it below, as it is tight to the the temporary definition of `_ ContextManager`.